.gitignore

```diff
@@ -4,6 +4,8 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
+.DS_Store
```

Comment:
@ondrae Sharing because you might find this useful! You can add files to your local global git ignore (say, create a file at ~/.gitignore_global) and then they'll be ignored from project-to-project. To register the global gitignore, then run `git config --global core.excludesfile '~/.gitignore_global'`.

Here's the contents of mine:
```
.idea
.DS_Store
```

Think it's fine to include in here, too.